includes/manage_newspaper.inc

```diff
@@ -19,10 +19,139 @@ require_once dirname(__FILE__) . '/utilities.inc';
  */
 function islandora_newspaper_manage_newspaper(AbstractObject $object) {
   drupal_set_title($object->label);
+  return drupal_get_form('islandora_newspaper_empty_issue_date_form', $object);
+}
+
+/**
+ * Form to add dates missing from newspaper issues.
+ *
+ * @param array $form
+ *   The form object.
+ * @param array $form_state
+ *   The form state
+ * @param AbstractObject|NULL $object
+ *   The parent of the issues missing dates or NULL for all issues.
+ *
+ * @return array
+ *   Renderable array of the form.
+ */
+function islandora_newspaper_empty_issue_date_form($form, &$form_state, $object) {
+  $empty_issues = islandora_newspaper_get_empty_dates($object);
+  if (count($empty_issues) > 0) {
+    $fields = array();
+    $form = array();
+    $form['dates_table'] = array(
+      '#type' => 'fieldset',
+      '#title' => t('Issues missing a dateIssued field.'),
+      '#description' => t('These newspapers will not appear in the browse listing if you use the "Order by date" setting. Add a date in the format "YYYY-MM-DD" and save the changes.'),
+    );
+    foreach ($empty_issues as $issue) {
+      $issue_obj = islandora_object_load($issue['pid']);
+      if ($issue_obj) {
+        $escaped_pid = str_replace(':', '_', $issue['pid']);
+        $escaped_pid = $issue['pid'];
```

Comment:
this looks odd should, probably don't need line 51 and 52 probably just one or the other?